Framework/DataObjects/inc/MantidDataObjects/TableColumn.h

```diff
@@ -1,31 +1,17 @@
 #ifndef MANTID_DATAOBJECTS_TABLECOLUMN_H_
 #define MANTID_DATAOBJECTS_TABLECOLUMN_H_
 
-//----------------------------------------------------------------------
-// Includes
-//----------------------------------------------------------------------
-
-#include "MantidAPI/Column.h"
-#include "MantidKernel/Logger.h"
-
-#include <vector>
-#include <typeinfo>
-#include <stdexcept>
+#include <boost/numeric/conversion/cast.hpp>
 #include <boost/shared_ptr.hpp>
 #include <limits>
-#include <boost/type_traits.hpp>
-#include <boost/numeric/conversion/cast.hpp>
-
-namespace Mantid {
+#include <sstream>
+#include <vector>
 
-//----------------------------------------------------------------------
-// Forward declarations
-//----------------------------------------------------------------------
+#include "MantidAPI/Column.h"
 
+namespace Mantid {
```

Comment:
The changes were indeed for another PR https://github.com/mantidproject/mantid/pull/19928, but because that has been merged the files no longer show changes